addons/danger/scripts/lambs_danger.fsm

```diff
@@ -1232,10 +1232,10 @@ class FSM
                          "[_this,_target] call lambs_danger_fnc_shareInformation; " \n
                          "" \n
                          "// change formation " \n
-                         "group _this setFormation (group _this getVariable [""lambs_danger_dangerFormation"",formation _this]);" \n
+                         "(group _this) setFormation (group _this getVariable [""lambs_danger_dangerFormation"",formation _this]);" \n
                          "" \n
-                         "// execute code " \n
-                         "[_this,units group _this] call ((group _this) getVariable [""lambs_danger_dangerCode"", {}]);" \n
+                         "// call event system" \n
+                         "[""lambs_danger_onContact"", [_this, group _this, units _this]] call lambs_danger_fnc_eventCallback;" \n
```

Comment:
@nk3nny do we need re-assessment to remove the call for `dangerCode` or can it be directly removed?

Reply:
I did not found any reference  to the variable in code

Reply:
Hmm. Every tweak of the FSM adds an odd overhead to the process. I suspect this FSM is different from the one I run in my pull request? One line of code is in any case no big thing. 

In any case. "dangerCode" was _only_ referenced on that point in the danger.fsm. It is only run when the group leader formally calls 'contact'

REACT (individuals hide) --> CONTACT (Group leader) --> ASSESS (Group leader) --> MISC TACTICAL ACTIONS (entire group)

There is a delay between REACT and CONTACT based on the squad leaders individual situation. 
There is a delay between CONTACT and ASSESS based on groups timing scheme-- 30-60 seconds 
There is delay between ASSESS and TACTICAL ACTIONS based on squad leaders timing scheme

If there are no potential TACTICAL ACTIONS  (as found by group leader ASSESS).  Then the group leader will continue to attempt periodic ASSESS. 

CONTACT is only called on 'first contact' or after 6 minutes without danger mode active.